playwright.config.ts

```diff
@@ -0,0 +1,45 @@
+import { PlaywrightTestConfig, devices } from "@playwright/test";
+
+const opts = {
+  // launch headless on CI, in browser locally
+  headless: !!process.env.CI || !!process.env.PLAYWRIGHT_HEADLESS,
+  collectCoverage: false, // not possible in Next.js 12
+  executablePath: process.env.PLAYWRIGHT_CHROME_EXECUTABLE_PATH,
```

Comment:
Out of curiosity, what is a scenario for a custom executable path?

Reply:
I think it just been hanging around there since the start for no good reason at all :)

Reply:
I'll try to remove it and see if something breaks 😅